_pages/algorithm-development/advanced-algorithm-development/local-development.md

```diff
@@ -25,9 +25,13 @@ Also be sure to keep your dependencies identical, down to the minor (or even the
 
 To help you identify the exact language and dependency versions, we've put together [code you can run inside your Algorithm to examine the environment and installed packages](list-packages/) -- and even to [help set up your virtual environment](list-packages/#creating-a-local-python-virtualenv-to-emulate-the-algorithms-environment).
 
+<div markdown="1">
```

Comment:
Adding the wrapping div prevents the `.syn-article` padding/margins from messing with the `.syn-alert` styles (Trying to add an exception via css requires a whole bunch of weirdness, this is much simpler.)